### DISCUSSION TITLE
CellCulture.txt example does not work

### DISCUSSION THREAD

**Discussion by kofideh** [topas-nbio/TOPAS-nBio#43] [Simulation-question] [answered]
### Simulation question

Hello:
When attempt to run the CellCulture.txt example in examples/geometry/cells, I get the error below. Does anyone have this experience? Any suggested solutions?

Thanks

-------- WWWW ------- G4Exception-START -------- WWWW -------
*** G4Exception : GeomVol1002
      issued by : G4PVPlacement::CheckOverlaps()
Overlap with volume already placed !
          Overlap is detected for volume MyCulture/Cell1:1 (G4Orb) with MyCulture/Cell0:0 (G4Orb)
          overlap at local point (0.00062449,0.000585965,0.000192178) by 3.44235 um  (max of 282 cases)
NOTE: Reached maximum fixed number -1- of overlaps reports for this volume !
*** This is just a warning message. ***
-------- WWWW -------- G4Exception-END --------- WWWW -------

**** Finding new position for volume Cell : 1 ****
^C
Session terminated.

-------- EEEE ------- G4Exception-START -------- EEEE -------
*** G4Exception : UI0001
      issued by : G4UIterminal::SignalHandler()
KeyboardInterrput with Ctrl-C
*** Fatal Exception *** core dump ***
 **** Track information is not available at this moment
 **** Step information is not available at this moment

-------- EEEE -------- G4Exception-END --------- EEEE -------

*** G4Exception: Aborting execution ***
/home/kofi/shellScripts/topas: line 6: 45055 Aborted                 (core dumped) ~/Applications/TOPAS/OpenTOPAS-install/bin/topas $1

### TOPAS-nBio category

Examples

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Attestation

- [X] I testify that I have fully read the bug reporting guideline beforehand.

**Comment by jschuemann** [comment]
The exception message is just a warning, as it says:
*** This is just a warning message. ***

The code tries to place the cells, checks for overlap with already placed cells, and if they overlap, exists, it:
**** Finding new position for volume Cell : 1 ****

Depending on the number of cells and volume size, this may take a long time. You can change how often you see this message if you want to follow the placements.

**Reply by kofideh** [reply]
Thank you for your response and suggestion, @jschuemann. The problem is, even after I set i:Ge/MyCulture/NumberofCells = 2, topas never concludes. I left it to run for 8 hours and it is still running. The only time topas concludes is for i:Ge/MyCulture/NumberofCells = 1. All other times, it seems to enter an infinite loop. I am using the current version of topas nbio, so it's puzzling why this is happening.

**Reply by JoseRamosMendez** [reply, marked as answer]
This could happen if the size of the envelope geometry is rather small so that the algorithm will take a lot of iterations, more likely falling very close to any previously accepted cell. If the space is too tight, you could instead position the cells by hand.

**Reply by kofideh** [reply]
Thanks, @JoseRamosMendez. I just positioned the cells manually as you suggested.
